frontend/src/components/RouteInputContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { IonDatetime, IonDatetimeButton, IonModal } from "@ionic/react";
+
+const RouteInputContainer: React.FC = () => (
+
+    <div>
+        <form style={{ marginBottom: 20 }}>
```

Comment:
You keep using absolute measures in this component. This is bad practice as this wouldn't adapt to the screen size.